tests/test_layout.py

```diff
@@ -352,10 +352,12 @@ def test_focus_directional_2x2grid(hlwm, client_focused, direction):
             .expect_stderr('No neighbour')
 
 
-def test_smart_window_surroundings(hlwm, x11):
+@pytest.mark.parametrize("border_width", [0, 7])
+def test_smart_window_surroundings(hlwm, x11, border_width):
     hlwm.call('set_layout vertical')
     hlwm.call('set frame_gap 0')
-    hlwm.call('set window_border_width 0')
+    hlwm.call('attr theme.minimal.border_width 0')  # only until #806 is merged
+    hlwm.call(f'set window_border_width {border_width}')
     hlwm.call('set frame_border_width 0')
     hlwm.call('set frame_padding 0')
     hlwm.call('set smart_window_surroundings on')
```

Comment:
In #803 you said that this test did not test the border width here. Should we add that now, so regressions are detected?
Checking that no border is drawn around the single window.

Reply:
This happens already in line 373 and 374. However, before this PR and before #806, there was really no border drawn because:
  * the usual border width was set to 0
  * smart_window_surroundings was not respected, so the usual border width was applied in line 372.

Since I now also set the window border width to 7, the tests in line 373 and 374 will fail on a regression.

Reply:
I've updated the coded and now:
* I also test different values for `theme.minimal.border_width`
* so we know that really the 'minimal decoration' is used for the client here
* so we are independent from PR #806 (the correct initial value is tested there and not here anyway)

Reply:
Awesome, thank you!